.github/workflows/nudgebot.yaml

```diff
@@ -0,0 +1,37 @@
+name: Label inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == "pantsbuild"
```

Comment:
This makes testing in forks annoying, because you have to change it or comment it out to get the workflow running?